Gems/PhysX/Common/3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+include(FetchContent)
+
+FetchContent_Declare(
+    v-hacd
+    URL          https://github.com/kmammou/v-hacd/raw/refs/tags/v4.1.0/include/VHACD.h
+    URL_HASH     MD5=19397dad8ffed62b1c6a24c404a3cd41
+    DOWNLOAD_NO_EXTRACT TRUE
+    DOWNLOAD_NO_PROGRESS TRUE
+    TLS_VERIFY TRUE
+)
+
+FetchContent_MakeAvailable(v-hacd)
+
+FetchContent_GetProperties(v-hacd SOURCE_DIR V_HACD_SOURCE_DIR)
+add_library(3rdParty::v-hacd IMPORTED INTERFACE GLOBAL ${V_HACD_SOURCE_DIR}/VHACD.h)
+target_include_directories(3rdParty::v-hacd SYSTEM INTERFACE ${V_HACD_SOURCE_DIR})
```

Comment:
consider using `ly_target_include_system_directories(xxxxx)` instead of `target_include_directories(xxxxx SYSTEM xxxx)` since that has macros behind it to work in situations where the compiler doesn't fully support it.  It has the syntax
```cmake
ly_target_include_system_directories(TARGET 3rdParty::v-hacd INTERFACE ${V_HACD_SOURCE_DIR})
```
so like, without the keyword `SYSTEM` and with the new keyword `TARGET`.